lib/classes/cartItems_box.dart

```diff
@@ -58,6 +58,7 @@ class CIBox {
       aquantity.id = aq.id;
       aquantity.value = aq.value;
       aquantity.metric = aq.metric;
+      if (aq.qty > 1) aq.qty = 1;
```

Comment:
@Rahul-Vijay is this for handling the qty when it is 1?

Reply:
- This function runs when 1st time an object is added which doesn't exist in cart.

When you add and then remove. Then add again, qty would be 2, or 3 or 4 depending on number of times you pressed on minus. So if it is greater than 1, means it was previously added then removed. So we reset it back to 1.